modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
 [id='proc-customize-rhdh-sidebar-menuitems_{context}']
 = Customizing the sidebar menu items for your {product-short} instance
 
-The sidebar menu in {product} consists of two main parts:
+The sidebar menu in {product} consists of two main parts that you can configure:
 
-* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
-
-* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
+* *Dynamic plugin menu items*: Your preferences and your active plugins define dynamically one part of the sidebar menu.
+* *Main menu items*: The core navigation structure of sidebar is static.
 
 .Procedure
 
-. Customize the main menu items using the following steps:
+. To configure *Dynamic plugin menu item*, add the following content to your `{my-app-config-file}` file:
```

Comment:
Afterthoughts for minimalism and clarity: 

Consider being more specific here:

```suggestion
. To configure *Dynamic plugin menu item*, add a `menuItems` section for your _<plugin_name>_ plugin to your `{my-app-config-file}` file:
```

Then, consider removing the second admonition (`menuItems: # <2>`), and renumbering the subsequent admonitions.